web_src/js/features/admin/common.js

```diff
@@ -84,7 +84,7 @@ export function initAdminCommon() {
     hideElem($('.oauth2_use_custom_url_field'));
     $('.oauth2_use_custom_url_field input[required]').removeAttr('required');
 
-    if ($('#oauth2_use_custom_url').is(':checked')) {
+    if ($('#oauth2_use_custom_url')[0].checked) {
```

Comment:
Same as below. why not directly replace it with `querySelector('#oauth2_use_custom_url').checked` ?

Since the changed code needs to be tested, I do not think it's worth to use 2 steps to change the code.

Reply:
I only partially tested this one as I wasn't sure under which condition this element shows. So what I did was unhide it by removing the class and checking the expression.

Reply:
Prefer to not introduce unrelated changes.